lrv/database/migrations/2019_03_06_064253_add_tracking_source_to_tracks.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddTrackingSourceToTracks extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('tracks', function($table) {
+            DB::statement('SET SQL_MODE = "ALLOW_INVALID_DATES";');
```

Comment:
The `tracks` table has a timestamp column definition that reads
`point_time timestamp NOT NULL DEFAULT '0000-00-00 00:00:00',`
`0000-00-00 00:00:00` is not a valid mysql timestamp value
This is preventing the migration from applying cleanly on my mysql server.
That is the reason for having this extra line here
`DB::statement('SET SQL_MODE = "ALLOW_INVALID_DATES";');`